src/tests/yamlCompletion.test.ts

```diff
@@ -4,6 +4,8 @@
  */
```

Comment:
would the changes work for nested "States" such as in Map and Parallel state?

if so, can we add a test for it?